### PR TITLE
Multi paths support

### PR DESCRIPTION
**Pull request by tkaczmarzyk**

### CODE REVIEW

**Comment by jradlica** [comment]
From CI: 
```
Results :

Tests in error: 
  filtersAccordingToInterfaceWithOrMultiPathVars_secondPath(net.kaczmarzyk.e2e.annotated.AnnotatedSpecInterfaceOrE2eTest): Request processing failed; nested exception is net.kaczmarzyk.spring.data.jpa.web.InvalidPathVariableRequestedException: Requested path variable {nickName} is not present in Controller request mapping annotations
  filtersAccordingToInterfaceWithOrMultiPathVars_firstPath(net.kaczmarzyk.e2e.annotated.AnnotatedSpecInterfaceOrE2eTest): Request processing failed; nested exception is net.kaczmarzyk.spring.data.jpa.web.InvalidPathVariableRequestedException: Requested path variable {lastName} is not present in Controller request mapping annotations
  resolvesPathVariableValue_requestMapping_multi_paths_first(net.kaczmarzyk.spring.data.jpa.web.WebRequestProcessingContextTest): Requested path variable {employeeId} is not present in Controller request mapping annotations
  resolvesPathVariableValue_requestMapping_multi_paths_second(net.kaczmarzyk.spring.data.jpa.web.WebRequestProcessingContextTest): Requested path variable {employeeId} is not present in Controller request mapping annotations
  testMethodUsingMultiPathVariableFromClass(net.kaczmarzyk.spring.data.jpa.web.SimpleSpecificationResolverPathVariablesTest): Requested path variable {employeeId} is not present in Controller request mapping annotations

Tests run: 952, Failures: 0, Errors: 5, Skipped: 0
```

File: src/main/java/net/kaczmarzyk/spring/data/jpa/web/annotation/MissingPathVarPolicy.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+ * limitations under the License.
+ */
+package net.kaczmarzyk.spring.data.jpa.web.annotation;
+
+/**
+ * @author Tomasz Kaczmarzyk
+ * @author Jakub Radlica
+ * @author Konrad Hajduga (Tratif sp. z o.o.)
+ */
+public enum MissingPathVarPolicy {
```
Comment (tkaczmarzyk) [inline src/main/java/net/kaczmarzyk/spring/data/jpa/web/annotation/MissingPathVarPolicy.java:23]: add a short javadoc explaining the purpose (similarly as in the readme (including my comments), but can be shorter)

File: src/main/java/net/kaczmarzyk/spring/data/jpa/web/annotation/Spec.java [hunk 2]
```diff
@@ -101,12 +101,19 @@
      * </p>
      */
     String path();
-    
+
+    /**
+     * Attribute is used to specify behaviour on missing path variable.
+     * Missing param policy should be set to IGNORED when controller endpoint contains multiple paths with path variables.
```
Comment (tkaczmarzyk) [inline src/main/java/net/kaczmarzyk/spring/data/jpa/web/annotation/Spec.java:107]: with different path variables

File: src/main/java/net/kaczmarzyk/spring/data/jpa/web/annotation/Spec.java [hunk 3]
```diff
@@ -101,12 +101,19 @@
...
+     * See example in comments of the following <a href="https://github.com/tkaczmarzyk/specification-arg-resolver/pull/93">pull-request</a>
```
Comment (tkaczmarzyk) [inline src/main/java/net/kaczmarzyk/spring/data/jpa/web/annotation/Spec.java:108]: Link to the readme section instead ?

**Pull request merged**